### DISCUSSION TITLE
Welcome to GitHub Hackathons Discussions!

### DISCUSSION THREAD

**Discussion by mishmanners** [github/hackathons#1] [General]
![Welcome](https://user-images.githubusercontent.com/36594527/117618914-f56fba80-b1b1-11eb-8a86-73871d594442.png)

# 👋 Hey Hackathon organizers! </p>

Discussions are a great place to connect with other awesome hackathon organizers and managers.

We hope you:
  * Asks questions you need answers too
  * Share ideas about organizing hackathons, a new organizer's tool, or whatever you want
  * Tell us your tips and tricks, and learn from others
  * Engage with other community members
  * Welcome others and are open-minded. Remember this is a community we build together 💪

Let's get started! Pick a category and open the Discussion:

### 🔮 [Tips and Tricks](https://github.com/MishManners/GitHub-Hackathons/discussions/categories/tips-and-tricks)
This is for tips and tricks. Feel free to discuss the tips and tricks outlined in the [Tips and Tricks documentation](https://github.com/MishManners/GitHub-Hackathons/blob/main/TipsandTricks.md).
If you'd like to share or discuss something new, feel free! If your tip or trick gets a lot of attention we might even add it into the documentation!

### 📚 [Resources](https://github.com/MishManners/GitHub-Hackathons/discussions/categories/resources)
A source of truth. Not really, just resources for organizers to share. Check out the [Resources documentation](https://github.com/MishManners/GitHub-Hackathons/blob/main/Resources.md), discuss it, comment on it. If there's something you think would be valuable, discuss it here, or add it to the documentation and [open a pull request](https://github.com/MishManners/GitHub-Hackathons/pulls). Make sure you follow the contribution guidelines detailed in the [Resources file](https://github.com/MishManners/GitHub-Hackathons/discussions/categories/resources) if you'd like to add a resource.

### 💡 [Ideas](https://github.com/MishManners/GitHub-Hackathons/discussions/categories/ideas)
Share ideas you have; new ways to run a hackathon, something that worked really well during your hackathon, or a new tool you think would be cool. Be nice to one another, comment, and give feedback, but don't shut people down.

### 🙏 [Q&A](https://github.com/MishManners/GitHub-Hackathons/discussions/categories/q-a)
Is there something you want to know? Ask it here. Let's see if the community knows the answer.

Good luck and looking forward to engaging and connecting with you.

**Comment by benksmillie** [comment]
This is an awesome resource - 🙏🏼 for the work pulling it together @MishManners

**Reply by mishmanners** [reply]
My pleasure. Glad you like it!
